core/auth/src/test/java/software/amazon/awssdk/auth/signer/Aws4SignerTest.java

```diff
@@ -156,6 +156,48 @@ public void xAmznTraceId_NotSigned() throws Exception {
                           "Signature=581d0042389009a28d461124138f1fe8eeb8daed87611d2a2b47fd3d68d81d73");
     }
 
+    /**
+     * Multi-value headers should be comma separated.
+     */
+    @Test
+    public void testMultiValueHeadersAreCommaSeparated() throws Exception {
```

Comment:
Nit, `test` seems a bit redundant. We don't have a consistent naming pattern for unit tests in v2, but most of them follow similar format of `methodUnderTest_when_expectedBehavior` I think

Reply:
Ack. Remnants of a JUnit 3 naming convention. I was trying to be mostly consistent with other unit tests in this class which did use `test...`, but let me update to use the more standard format.